extensions/amp-consent/amp-consent.md

```diff
@@ -137,6 +146,10 @@ current user to the 3rd party vendor extensions.
 
 Unlike consent state, this `shareData` is not persisted in client side storage.
 
+#### consentRequired
```

Comment:
worth adding a link to the `geoOverride` section, where `consentRequired` makes more sense.